internal/stores/gcs.go

```diff
@@ -68,21 +67,16 @@ func (s *GCSStore) GetFsys() (afero.Fs, error) {
 
 // Upload generates the metadata, writes it s.fsys and uploads the file to the GCS bucket
 func (s *GCSStore) Upload(ctx context.Context, objects ...string) error {
-	var multErr error
 	for _, o := range objects {
 		logger.V(2).Infof("Object: %s\n", o)
 		f, err := s.fsys.Open(o)
 		if err != nil {
 			return err
 		}
 		defer f.Close()
-
-		// cleanupFn is function that can be called if
-		// uploading to s3 fails. cleanupFn deletes the cfile
-		// so that we don't retain a cfile without a corresponding binary
-		cleanupFn, err := WriteMetadataToFsys(s, o, f)
+		_, err = f.Seek(0, io.SeekStart)
```

Comment:
We no longer need to reset the offset here, right?

Reply:
It doesn't hurt anything to have it, I'm not quite sure if it's strictly needed.